contracts/src/v0.8/automation/upkeeps/LinkAvailableBalanceMonitor.sol

```diff
@@ -33,7 +34,7 @@ interface ILinkAvailable {
 ///  this is a "trusless" upkeep, meaning it does not trust the caller of performUpkeep;
 /// we could save a fair amount of gas and re-write this upkeep for use with Automation v2.0+,
 /// which has significantly different trust assumptions
-contract LinkAvailableBalanceMonitor is ConfirmedOwner, Pausable, AutomationCompatibleInterface {
+contract LinkAvailableBalanceMonitor is ConfirmedOwner, Pausable, AccessControl, AutomationCompatibleInterface {
```

Comment:
Do we need both `ConfirmedOwner` _and_ `AccessControl`? I think the latter is a "super set" of the former.

Reply:
I thought the same, but after some testing it seems ConfirmedOwner is needed by Pausable. If the contract doesn't implement ConfirmedOwner it would fail. I'll be testing this a bit more though.